middleware/errors/errors.go

```diff
@@ -51,6 +52,8 @@ func Middleware(entry *logrus.Entry) graphql.RequestMiddleware {
 			} else if err != nil {
 				contextEntry.WithError(err).Warnf("GQL Request Client Error: %v err = %+v", err.Extensions["general_error"], err)
 			}
+
+			localHub.RecoverWithContext(ctx, err)
```

Comment:
This catches GQL errors like this: https://sentry.io/organizations/dapperlabs/issues/3453822061/?project=6398442&query=&statsPeriod=14d